.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -841,6 +841,71 @@ stages:
             baseImage: $(baseImage)
             command: "CheckBuildLogsForErrors"
 
+- stage: exploration_tests_windows
+  condition: and(succeeded(), eq(variables['isScheduledBuild'], 'False'), eq(variables['runExplorationTests'], 'True'))
+  dependsOn: build_windows
+
+  jobs:
+  - template: steps/update-github-status-jobs.yml
+    parameters:
+      jobs: [ExplorationTest]
+
+  - job: generator
+    pool:
+      vmImage: ubuntu-18.04
+    steps:
+      - bash: |
+          matrix="{"
+          for explorationTestUseCase in "debugger" "continuousProfiler"; do
+            for explorationTestName in "eshoponweb" "protobuf" "cake" "swashbuckle" "paket" ; do
+              name="${explorationTestUseCase//./_}_${explorationTestName}"
+              matrix="${matrix} '${name}':"
+              matrix="${matrix} { 'explorationTestUseCase': '${explorationTestUseCase}', "
+              matrix="${matrix} 'explorationTestName': '${explorationTestName}' },"
+            done
+          done
+          matrix="${matrix::-1} }"
+          echo "##vso[task.setVariable variable=values;isOutput=true]${matrix}"
+        name: mtrx
+
+  - job: ExplorationTest
+    timeoutInMinutes: 100
+    dependsOn: generator
+    strategy:
+      matrix: $[ dependencies.generator.outputs['mtrx.values'] ]
+    pool:
+      vmImage: windows-2019
+
+    # Enable the Datadog Agent service for this job
+    services:
+      dd_agent: dd_agent
+
+    steps:
+    - template: steps/install-dotnet-sdks.yml
+      parameters:
+        includeX86: true
+    - template: steps/restore-working-directory.yml
+
+    - script: tracer\build.cmd RunExplorationTests_$(explorationTestUseCase) -ExplorationTestName $(explorationTestName)
+      displayName: RunExplorationTest
+
+    - task: PublishTestResults@2
+      displayName: publish test results
+      inputs:
+        testResultsFormat: VSTest
+        testResultsFiles: tracer/build_data/results/**/*.trx
+      condition: succeededOrFailed()
```

Comment:
Lets remove this task, as we're not generating any test results, so it's just generating warnings 🙂

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/2089/commits/cb2d0530bf02d495124abe0c217852d940247f40